README.md

```diff
@@ -1,24 +1,90 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Car Rentals Application UI
```

Comment:
- [ ]  Your app has no tests, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍
- [ ] Please, consider adding some tests in order to endure your app's integrity.